src/components/Layout/Layout.scss

```diff
@@ -1,8 +1,20 @@
+@import 'styles/utilities/breakpoints.scss';
```

Comment:
Galen added some logic to this file in his PR. Can you remove this for now? Otherwise there is going to be a conflict

Reply:
I can do that but it will throw an error in my PR. I think it might be better for him to do it with his changes unless our PR is going to be merged right away.

Reply:
well technically if you merge main you'll see his changes cause he already pushed it